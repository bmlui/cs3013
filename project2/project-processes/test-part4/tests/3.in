sleep 30 &
sleep 20 &
sleep 10 &
kill 2
kill 5
jobs
exit

