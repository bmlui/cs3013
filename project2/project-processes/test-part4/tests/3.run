./wshell < tests/3.in 
