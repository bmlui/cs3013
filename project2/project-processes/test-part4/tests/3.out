test-part4$ sleep 30 &
[1]
test-part4$ sleep 20 &
[2]
test-part4$ sleep 10 &
[3]
test-part4$ kill 2
test-part4$ kill 5
wshell: no such background job: 5
test-part4$ jobs
1: sleep 30
3: sleep 10
test-part4$ exit
